source/scale/scale-to-50000-users.rst

```diff
@@ -15,15 +15,39 @@ This page describes the Mattermost reference architecture designed for the load
 Requirements
 ------------
 
-+------------------------+----------+------------------+----------------+------------------+
-| **Resource Type**      | **vCPU** | **Memory (GiB)** | **# of Nodes** | **AWS Instance** |
-+========================+==========+==================+================+==================+
-| Mattermost Application | 8        | 16               | 5              | c6i.2xlarge      |
-+------------------------+----------+------------------+----------------+------------------+
-| RDS Writer             | 8        | 64               | 1              | db.r6g.2xlarge   |
-+------------------------+----------+------------------+----------------+------------------+
-| RDS Reader             | 8        | 64               | 4              | db.r6g.2xlarge   |
-+------------------------+----------+------------------+----------------+------------------+
+Without Elasticsearch
+~~~~~~~~~~~~~~~~~~~~~
+
++------------------------+----------------+-------------------+
+| **Resource Type**      | **# of Nodes** | **AWS Instance**  |
++========================+================+===================+
+| Mattermost Application | 4              | c7i.2xlarge       |
++------------------------+----------------+-------------------+
+| RDS Writer             | 1              | db.r7g.2xlarge    |
++------------------------+----------------+-------------------+
+| RDS Reader             | 3              | db.r7g.2xlarge    |
++------------------------+----------------+-------------------+
+| Elasticsearch Node     | 0              | r6g.xlarge.search |
++------------------------+----------------+-------------------+
+| Proxy                  | 1              | m7i.4xlarge       |
++------------------------+----------------+-------------------+
+
+With Elasticsearch
+~~~~~~~~~~~~~~~~~~~
+
++------------------------+----------------+-------------------+
+| **Resource Type**      | **# of Nodes** | **AWS Instance**  |
++========================+================+===================+
+| Mattermost Application | 5              | c7i.2xlarge       |
++------------------------+----------------+-------------------+
+| RDS Writer             | 1              | db.r7g.2xlarge    |
++------------------------+----------------+-------------------+
+| RDS Reader             | 4              | db.r7g.2xlarge    |
++------------------------+----------------+-------------------+
+| Elasticsearch Node     | 2              | r6g.xlarge.search |
++------------------------+----------------+-------------------+
+| Proxy                  | 1              | m7i.4xlarge       |
++------------------------+----------------+-------------------+
```

Comment:
The data that we have is exactly this. But should we repeat the 4 app nodes and 4 DB nodes test *with* Elasticsearch so that we don't need to bump the number of app and DB nodes in the Elasticsearch section?

Reply:
@nab-77 - Without ES section was removed ^